msg/MotionReadyEnum.msg

```diff
@@ -61,4 +61,7 @@ string NOT_READY_OTHER_TRAJ_MODE_ACTIVE_STR="Another trajectory mode is already
 uint16 NOT_READY_NOT_CONT_CYCLE_MODE=115
 string NOT_READY_NOT_CONT_CYCLE_MODE_STR="Continuous cycle mode not set. Please set the cycle mode to continuous"
 
+uint16 NOT_READY_MAJOR_ALARM=116
+string NOT_READY_MAJOR_ALARM_STR="The controller has an active Major Alarm that cannot be reset. Check teach pendant"
```

Comment:
Let's be kind :)

```suggestion
string NOT_READY_MAJOR_ALARM_STR="The controller has an active Major Alarm that cannot be reset. Please check the teach pendant"
```

PS: isn't it technically the *programming* pendant?

Reply:
> Let's be kind :)

NEIN! You vill check zee pendant! Do not dilly dally.

> PS: isn't it technically the programming pendant?

Meh... interchangeable terminology.

Reply:
Updated to be kinder 😄 

I kept it as "teach pendant" to be consistent with other `MotionNotReadyCode` strings.